CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+# [VuFind Developer Manual](https://vufind.org/wiki/development)
+
+## Prerequisites
+
+ - [Community Code of Conduct](https://github.com/vufind-org/vufind/CODE_OF_CONDUCT.md)
+ - [Coding Standards](https://vufind.org/wiki/development:coding_standards) - How to conform to VuFind's style.
+ - [Recommended Tools](https://vufind.org/wiki/development:recommended_tools) - Suggestions for successful VuFind development.
+ - [Making Pull Requests](https://vufind.org/wiki/development:making_pull_requests)
+ 
+## About VuFind
+
+ - [Architecture Notes](https://vufind.org/wiki/development:architecture) - Information on VuFind's design and dependencies.
```

Comment:
I wonder if less is more here, and we'd be better off just linking to the [top-level developers manual page](https://vufind.org/wiki/development) so we don't have to maintain so many links in multiple places.

Reply:
That's probably the easiest to maintain but it felt strange to me just having the Contributing guide be "go over here". Didn't see a precedent for that.